README.md

```diff
@@ -12,6 +15,7 @@ A library for avoiding TransactionTooLargeException during state saving and rest
 * [Limitations](#limitations)
 * [Testing](#testing)
 * [License](#license)
+* [Javadoc](https://jitpack.io/com/github/livefront/bridge/feature~hosted-javadoc-v1.1.0-g45d8d59-1/javadoc/index.html)
```

Comment:
Yeah, I agree. I am not finding a way to always link to the latest release... perhaps we could consider removing this link to the javadoc entirely, considering it is now included when you add this project via gradle? @byencho